packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -293,7 +296,14 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         </div>
 
         { isOpen && (
-          (onRenderContainer as any)({ ...this.props as any }, this._onRenderContainer)
+          (onRenderContainer as any)({
+            ...this.props,
+            onRenderList: onRenderList,
+            onRenderItem: onRenderItem,
+            onRenderOption: onRenderOption,
```

Comment:
I'm not quite sure about passing modified props down... but since it's easier than adding something new I think I would be fine with that.

@jspurlin Thoughts?

Reply:
We can also create a new type for these IRenderFunctions, if that's better

Reply:
We could do something like this for back-compat and to not change the props before passing them along:

```ts
interface IComboBoxRenderProps extends IComboBoxProps {
  currentOptions: IComboOption[];
  renderOverrides: {
    onRenderList: IRenderFunction<...>;
    ...
  };
}
```

but that might be more confusing.

Reply:
At least we are not changing the props, but merely copying them. At the end of the day, IComboBoxProps is just a type to the render functions, rather than the actual props.

I don't really have a strong opinion either way